assets/prototype/domain/eventEditor/tickets/ticketList/filterBar/useTicketListFilterState/index.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { useReducer, useEffect } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import filters from '../../../../../shared/predicates/tickets/filters';
+import { DisplayDates, ShowTickets, SortTickets } from '../../../../data/ticket/types';
+import { Ticket } from '../../../../data/types';
+
+enum ActionType {
+	SET_DISPLAY_TICKET_DATE = 'SET_DISPLAY_TICKET_DATE',
+	SET_SHOW_TICKETS = 'SET_SHOW_TICKETS',
+	SET_TICKETS_SORTED_BY = 'SET_TICKETS_SORTED_BY',
```

Comment:
maybe change this to match the name change for the enum?

```suggestion
	SET_SORT_TICKETS = 'SET_SORT_TICKETS',
```

Reply:
Done, thx.